<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sequel Fast</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
    <property name="verticalSpacing">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBoxLeft">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="pageTables_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>266</width>
         <height>468</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset>
         <normalon>:/icons/resources/element.svg</normalon>
        </iconset>
       </attribute>
       <attribute name="label">
        <string>Tables</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEditTables">
            <property name="styleSheet">
             <string notr="true">QLineEdit {padding: 4px; border-color: #ddd; border-width: 1px; border-radius: 6px}</string>
            </property>
            <property name="maxLength">
             <number>80</number>
            </property>
            <property name="placeholderText">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonFilterTables">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/resources/heart tick.svg</normaloff>:/icons/resources/heart tick.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonEditTables">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/resources/brush.svg</normaloff>:/icons/resources/brush.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonUpdateTables">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/resources/refresh square.svg</normaloff>:/icons/resources/refresh square.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listViewTables">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSchemas_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>266</width>
         <height>468</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset>
         <normalon>:/icons/resources/layer.svg</normalon>
        </iconset>
       </attribute>
       <attribute name="label">
        <string>Schemas</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEditSchemas">
            <property name="styleSheet">
             <string notr="true">QLineEdit {padding: 4px; border-radius: 6px}</string>
            </property>
            <property name="maxLength">
             <number>80</number>
            </property>
            <property name="placeholderText">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonFilterSchemas">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/resources/heart tick.svg</normaloff>:/icons/resources/heart tick.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonEditSchemas">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/resources/brush.svg</normaloff>:/icons/resources/brush.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonUpdateSchemas">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/resources/refresh square.svg</normaloff>:/icons/resources/refresh square.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listViewSchemas">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageConnections_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>264</width>
         <height>468</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset>
         <normalon>:/icons/resources/cloud connection.svg</normalon>
        </iconset>
       </attribute>
       <attribute name="label">
        <string>Connections</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEditConns">
            <property name="styleSheet">
             <string notr="true">QLineEdit {padding: 4px; border-radius: 6px}</string>
            </property>
            <property name="maxLength">
             <number>80</number>
            </property>
            <property name="placeholderText">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonNewConns">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/resources/add square.svg</normaloff>:/icons/resources/add square.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonEditConns">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/resources/brush.svg</normaloff>:/icons/resources/brush.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listViewConns">
          <property name="styleSheet">
           <string notr="true">QListView {padding: 6px}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_connection"/>
   <addaction name="actionNew_schema"/>
   <addaction name="actionNew_table"/>
   <addaction name="actionNew_query"/>
   <addaction name="separator"/>
   <addaction name="actionInformation"/>
   <addaction name="actionUsers"/>
   <addaction name="actionStatistics"/>
   <addaction name="actionVariables"/>
   <addaction name="actionProcesses"/>
   <addaction name="separator"/>
   <addaction name="actionTile"/>
   <addaction name="actionCascade"/>
  </widget>
  <action name="actionNew_connection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/cloud connection.svg</normaloff>:/icons/resources/cloud connection.svg</iconset>
   </property>
   <property name="text">
    <string>New connection</string>
   </property>
  </action>
  <action name="actionNew_schema">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/layer.svg</normaloff>:/icons/resources/layer.svg</iconset>
   </property>
   <property name="text">
    <string>New schema</string>
   </property>
  </action>
  <action name="actionNew_table">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/copy.svg</normaloff>:/icons/resources/copy.svg</iconset>
   </property>
   <property name="text">
    <string>New table</string>
   </property>
  </action>
  <action name="actionNew_query">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/receipt.svg</normaloff>:/icons/resources/receipt.svg</iconset>
   </property>
   <property name="text">
    <string>New query</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/information.svg</normaloff>:/icons/resources/information.svg</iconset>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/setting (1).svg</normaloff>:/icons/resources/setting (1).svg</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/login.svg</normaloff>:/icons/resources/login.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUsers">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/profile user.svg</normaloff>:/icons/resources/profile user.svg</iconset>
   </property>
   <property name="text">
    <string>Users</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/diagram.svg</normaloff>:/icons/resources/diagram.svg</iconset>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionVariables">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/shapes.svg</normaloff>:/icons/resources/shapes.svg</iconset>
   </property>
   <property name="text">
    <string>Variables</string>
   </property>
  </action>
  <action name="actionProcesses">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/bubble.svg</normaloff>:/icons/resources/bubble.svg</iconset>
   </property>
   <property name="text">
    <string>Processes</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/element.svg</normaloff>:/icons/resources/element.svg</iconset>
   </property>
   <property name="text">
    <string>Tile</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionCascade">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/square.svg</normaloff>:/icons/resources/square.svg</iconset>
   </property>
   <property name="text">
    <string>Cascade</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEditTables</tabstop>
  <tabstop>buttonFilterTables</tabstop>
  <tabstop>buttonEditTables</tabstop>
  <tabstop>buttonUpdateTables</tabstop>
  <tabstop>lineEditSchemas</tabstop>
  <tabstop>buttonFilterSchemas</tabstop>
  <tabstop>buttonEditSchemas</tabstop>
  <tabstop>buttonUpdateSchemas</tabstop>
  <tabstop>lineEditConns</tabstop>
  <tabstop>buttonNewConns</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
